File: project_flutter/src/main/java/com/example/project_flutter/entity/Location.java
package com.example.project_flutter.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;

/**
 * @author 西云研究所
 * @date 2021/10/18
 */
@Component
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Location {
    private long location_id;
    private String location_building;
    private String location_room;
}

File: untitled1/lib/project/entity/Location.dart
class Location{
  int location_id;
  String location_building;
  String location_room;

  Location({
    this.location_building,
    this.location_id,
    this.location_room,
});

  factory Location.fromJson(Map<String, dynamic>parsedJson){
    return new Location(
      location_building: parsedJson["location_building"],
      location_id: parsedJson["location_id"],
      location_room: parsedJson["location_room"],
    );
  }
}